core-meetings/2019-11-oss-meeting.md

```diff
@@ -0,0 +1,33 @@
+# Open Source Maintenance: November 2018 Core Meeting
+
+As stewards of React Native engineers worldwide, we've contemplated ways to improve alignment and visibility of the framework's direction, where community members and Facebook alike joined to discuss issues and propose responses.
+
+The community team is made up of volunteers who help maintain React Native, contribute to education materials, and improve the React Native developer's environment.
+
+## The core discussion
+
+This time, many members from both the community and the FB team met to talk about the Open Source repository and its maintenance.
+
+The first part of the conversation was about two aspects of the current release (0.57.x):
+
+- how the React sync commits land in master, and have impacted the releases
+- how to tackle the patch releases and cherry picks request, given that over time this is more complex and prone to merge conflicts (because of the speed at which master branch moves)
+
+During the discussion, the FB team also clarified a couple of points from their latest blogpost about [Open Source Roadmap](http://facebook.github.io/react-native/blog/2018/11/01/oss-roadmap), here reported for transparency sake:
```

Comment:
I think this was clarified by Eli ...

Reply:
Ok, but I don't like using "single person" statements in meeting notes; I've deliberately avoided "pointing fingers" in any way. If you feel strongly about saying that Eli was the one clarifying I can change it.